taca_ngi_pipeline/cli.py

```diff
@@ -119,13 +137,35 @@ def project(ctx, projectid, snic_api_credentials=None, statusdb_config=None, ord
                 add_user=list(set(add_user)),
                 fcid=fc_delivery,
                 **ctx.parent.params)
+        elif ctx.parent.params['cluster'] == 'dds': # Hard stage and deliver using DDS
+            if statusdb_config == None:
+                logger.error("--statusdb-config or env variable $STATUS_DB_CONFIG need to be set to perform DDS delivery")
+                return 1
+            load_yaml_config(statusdb_config.name)
+            if order_portal == None:
+                logger.error("--order-portal or env variable $ORDER_PORTAL need to be set to perform DDS delivery")
+                return 1
+            load_yaml_config(order_portal.name)
```

Comment:
This could be common to both grus and dds.

Reply:
True, I'll rewrite that 👍